dependencies.gradle

```diff
@@ -8,6 +8,7 @@ ext {
             awsLambdaJavaEvents  : "3.11.2",
             jacksonAnnotations   : "2.15.2",
             jacksonDatabind      : "2.15.2",
+            jacksonDataformatXml : "2.15.2",
```

Comment:
I think we don't need to specify each separate type of jackson here, they all use the same versions, so could you commonise :)